app/(app)/home.tsx

```
@@ -29,57 +41,110 @@ const HomeScreen = () => {
+            <InfoRow label="Last Name" value={profile?.last_name} />
+            <InfoRow label="Date of Birth" value={profile?.dob} />
           </View>
-
-          <View style={styles.userInfoItem}>
-            <Text style={styles.userInfoLabel}>Auth Provider:</Text>
-            <Text style={styles.userInfoValue}>
-              {authState.user?.app_metadata?.provider || "Not available"}
-            </Text>
+        </View>
+        {/* Assessment Information Section */}
+        // Update the Assessment Information Section to handle an array of
```

Biome (1.9.4)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)